k-distribution/src/main/scripts/lib/k-util.sh

```diff
@@ -73,14 +73,27 @@ fi
 
 execute () {
   (
-  if ${verbose}; then
-    set -x
-  fi
+  set +e
+
+  declare -a cmd
   if ${profile}; then
     TIMEFORMAT="%lR %lU %lS $*"
-    time "$@"
+    cmd=(time "$@")
```

Comment:
I would really prefer we continue to use the shell's time command rather than the one located at /usr/bin/time.

Reply:
Sure, no problem. I switched to make the code a bit cleaner, but I'm happy to revert for the sake of minimising the dependency surface